packages/core/src/routes/interaction/mfa.ts

```diff
@@ -121,6 +121,21 @@ export default function mfaRoutes<T extends IRouterParamContext>(
         { accountId, rpId: hostname, origin }
       );
 
+      // Update last used time
+      const user = await queries.users.findUserById(accountId);
+      await queries.users.updateUserById(accountId, {
+        mfaVerifications: user.mfaVerifications.map((mfa) => {
+          if (mfa.id !== verifiedMfa.id) {
+            return mfa;
+          }
+
+          return {
+            ...mfa,
+            lastUsedAt: new Date().toISOString(),
+          };
+        }),
+      });
```

Comment:
Could use a single query method? `updateUserMfaVerificationsById(userId, mfaId)`. Looks like no need to query user then update.

Reply:
created LOG-7236 to optimize.